### ISSUE TITLE
Schematic parity to PCB

### ISSUE THREAD

**Issue by db-ok** [db-ok/SofleChocWireless#4]
The PCB and the schematic in this repo are not in sync, so attempting to modify the schematic and have it reflected correctly in the PCB may result in problems on the PCB. As of right now, the PCB itself is perfectly fine and produces valid gerbers that can be sent for manufacturing, but for those wishing to customize the design of this board, they should be aware that the schematic would need to be brought into sync with the PCB first in order for schematic changes to be correctly reflected on the PCB.

**Comment by MMMikeM** [comment]
Hey @db-ok, I'm curious, what schematic changes will need to be made? I'm guessing here but

- additional LED
- Battery & power switch
- Removing the TRRS

Anything else?

**Comment by db-ok** [comment]
The main thing is that the way I developed the PCB was directly in the PCB editor, not by changing the schematic first, then using that to update the PCB. Because of this, I never updated the schematic with any of the things I added or modified directly on the PCB editor compared to the original Sofle Choc repo. You're correct, though, those items you listed will all be missing, and there are probably other things that I changed which need to be added/cleaned up/changed to get the schematic lined up with the PCB. I think the main thing would be giving each footprint in the PCB its own unique name, which is an issue I created by simply copy/pasting items around in the PCB, so multiple footprints have the same identifier. If that issue got fixed, I think you could just use the PCB to update the schematic to get them synced up. I may get around to fixing this myself one day, but if you just want to generate gerbers and make the keyboard, everything works for that as-is.

**Comment by db-ok** [comment]
@MMMikeM I had another look at the Kicad files, and the schematic parity issue is not as much trouble to resolve as I thought it would be. I created the `schematic_parity` branch with the changes that address this issue, and I have ordered some PCBs with the gerber files on that branch to make sure nothing went wrong just by fixing the schematic. I will plan to merge those changes to main and close this issue after I get the PCBs back from fabrication and have a chance to assemble and test them out.
